core/src/main/scala/Block.scala

```diff
@@ -30,6 +30,14 @@ object Block {
       valueLength: Int = 90
   ): Block =
     Block.fromSource(Source.fromURI(path.toURI), keyLength, valueLength)
+
+  def sort(block: Block): Block = {
+    val sortedRecords = block.records.sortBy(_.key)
+    new Block(sortedRecords)
+  }
```

Comment:
Nit: add new line between methods.
```suggestion
  }

```

Reply:
Sure